scss/civicrm/common/_block-shadows.scss

```diff
@@ -0,0 +1,55 @@
+/* stylelint-disable max-nesting-depth, selector-max-compound-selectors, selector-no-qualifying-type, selector-max-id */
```

Comment:
just as a comment. It seems we keep disabling the same rules again and again. We should either:

* Give up, because we don't have control of the CSS/HTML structure and disable these rules globally.
* Use a `.stylelintrc` file with these rules disabled and place these files in a specific folder within the project.
* Fix the structure of CiviCRM directly (not realistic)

Reply:
This is a requirement in Shoreditch: https://github.com/civicrm/org.civicrm.shoreditch/blob/master/CODING.md

I totally agree @reneolivo this does not look right, but we use linter in Bootstrap.

Reply:
If all the files need that rule at the top we can have a `.stylelintrc` file for the `civicrm/` folder only. That will apply the rule only to files under that folder.

Again, nothing to do here. Just a comment.